lib/iex/lib/iex/autocomplete.ex

```diff
@@ -349,7 +349,8 @@ defmodule IEx.Autocomplete do
 
   defp match_modules(hint, root) do
     get_modules(root)
-    |> :lists.usort()
+    |> Enum.sort()
+    |> Enum.uniq()
```

Comment:
You can use `Enum.sort() |> Enum.dedup()` as a slightly less expensive alternative to `uniq` on an already sorted data.

Reply:
@michalmuskala great advice!
I am applying the same principle here https://github.com/elixir-lang/elixir/pull/8467/files#diff-0a5aeba395ef65ae5e0061fc5b4bbf16R163

Reply:
I have reverted this other one though because it is completely fine to call functions in `:lists` when they don't have an equivalent in our List module. ;)